### PR TITLE
Update dutch translation

### PR DESCRIPTION
**Pull request by FaBjE**
I have updated the dutch translation. (Added missing parts, improved spelling/terms, etc)
Some translations are "too long" (indicated red) because they can't be translated any shorter in dutch...

I have not tested this on a soldering iron.

### CODE REVIEW

File: Translations/translation_NL.json [hunk 1]
```diff
@@ -1,207 +1,339 @@
...
+{
+ "languageCode": "NL",
+ "languageLocalName": "Nederlands",
+ "fonts": [
+  "ascii_basic",
+  "latin_extended"
+ ],
+ "tempUnitFahrenheit": false,
+ "messages": {
+  "SettingsCalibrationWarning": "Zorg ervoor dat te punt op kamertemperatuur is voor je verder gaat!",
```
Comment (KoopsF) [inline Translations/translation_NL.json:10]: "Zorg ervoor dat de punt op kamertemperatuur is voor je verder gaat!"

**Review by KoopsF — CHANGES_REQUESTED** [review]

File: Translations/translation_NL.json (hunk 1 above)
Comment (FaBjE) [reply Translations/translation_NL.json]: Good catch! Fixed it.

File: Translations/translation_NL.json [hunk 2]
```diff
@@ -1,207 +1,339 @@
...
+  "SettingsResetWarning": "Weet je zeker dat je de fabrieksinstellingen terug wil zetten?",
```
Comment (KoopsF) [inline Translations/translation_NL.json:11]: "Weet je zeker dat je de fabrieksinstellingen terug wilt zetten?"
Found another typo. Is this the best way of commenting on them?

Comment (FaBjE) [reply Translations/translation_NL.json]: I don't know if it is the best way, but it works for me :)
(Fixed it)

**Pull request merged**